itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,267 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
+    import "plugin-check-command"
+
+    command = [ PluginContribDir + "/thola-client", "check" ]
+
+    arguments += {
+        "--target-api" = {
+            required = true
+            value = "$thola_api_address$"
```

Comment:
Can this default to `$check_address$` or do I oversee something Thola-specific?

Reply:
The thola api address and the host ip address are two different addresses, so I don't think that this will work